Sharing an article documentation V1 (#117)
exoplatform/exo-documentation: docs/GettingStarted.rst

> .. _Sharing-news:

Reviewer: Shouldn't we rename also this ?

Author: I had the same idea but didn't know how to rename it. WDYT of "working-with-news"?

Reviewer: just "news" ?

Author: OK. I'll do it in the next commit.

|image162|

Editing a News
---------------

You can change the content of the article using the edit icon. 
The possible actions from the edit mode interface are `Update`, `Update and post` and `Cancel`. The buttons `Update` and `Update and post` are disabled until changes are made.
The `Update` action allows you to make changes in the article without reposting it unlike the `Update and post` action that allows to apply the changes in the article details and rise up the article's preview in the activity stream.

|image163|

When an article has been edited, the details view display the updated date and author besides initial information.

|image164|

Sharing a News
---------------

Once an article is posted, you will be allowed to share it in other spaces. The sharing action is possible from the article details page (and from the article's preview in the activity stream). 

> When you click on the share icon, a popup appears containing a reminder of the title of the article and allowing you to select the distribution space(s), add a description to be displayed in the activity that will be generated and validate the sharing action.

Reviewer: ```suggestion
When you click on the share icon, a popup appears containing a reminder of the title of the article and allowing you to select the destination space(s). You can also add a optional description which will be displayed in the sharing activity.
```

|image165|

>  ... note:: Sharing articles is only possible to spaces for the moment, you will be able to select only the spaces in which you are a member.

Reviewer: ```suggestion
 ... note:: Sharing articles is only possible to spaces for the moment. You will be able to select only the spaces in which you are a member.
```

 
Once the article is shared, a message confirming that the action has been completed is displayed with the list of the destination space(s).

|image166|

> You can check that the action has been done succesfully by connecting to the concerned spaces. The article preview will be available in the activity stream with the description if this field has been filled in.

Reviewer: ```suggestion
You can check that the action has been done successfully by connecting to the destination spaces. The article preview will be available in the activity stream with the description if this field has been filled in.
```
